Client/core/CScreenShot.cpp

```diff
@@ -48,8 +51,11 @@ void CScreenShot::InitiateScreenShot(bool bIsCameraShot)
 
     if (bIsCameraShot)
     {
-        // Set the screenshot path to camera gallery path
-        ms_strScreenDirectoryPath = PathJoin(GetSystemPersonalPath(), "GTA San Andreas User Files", "Gallery");
+        if (savePhotoInDocuments)
+        {
+            // Set the screenshot path to camera gallery path
+            ms_strScreenDirectoryPath = PathJoin(GetSystemPersonalPath(), "GTA San Andreas User Files", "Gallery");
+        }
```

Comment:
You **can** remove the braces

Reply:
Personally I like to have the brackets indicating a branch but if the mta team decides on to shorten them out, I can do that.

Reply:
I think it's personal preference, makes no difference in readability or when compiled afaik